### PR TITLE
1390-containedIn

### PR DESCRIPTION
**Pull request by TobiasNx**
See #1390. I added the  `containedIn`-array.
I transformed all 773 - Host Item Entry (R) with matching subfield $w .

### CODE REVIEW

**Review by dr0i — APPROVED** [review]
+1 but wouldn't that the point to solve the question first re redundance?

**Comment by TobiasNx** [comment]
I need to reset this PR since it needs needs to include #1397 too

File: src/main/resources/alma/fix/relatedRessourcesAndLinks.fix [hunk 1]
```diff
@@ -149,6 +149,25 @@ do list(path: "830??", "var": "$i")
   end
 end
 
+# containedIn
+# TODO: What is the difference between containedIn and isPartOf?
```
Comment (dr0i) [inline src/main/resources/alma/fix/relatedRessourcesAndLinks.fix:153]: Should this question be solved right now or do we want to postpone it ?

Comment (TobiasNx) [reply src/main/resources/alma/fix/relatedRessourcesAndLinks.fix:153]: Found some documentation on that: https://github.com/hbz/lobid-resources/issues/1400#issue-1347591143

`containedIn` is used with articles that are ressources depending on an umbrella publication as an journal issue or an anthology. The ressource is not published by itself.

`isPartOf`: is used with ressources that are published by themself but are part of an series, journal etc.

https://lobid.org/resources/api

**Comment by TobiasNx** [comment]
This ticket is replaced by #1402

**Pull request closed without merging**